File: Cash/cash.c
#include <stdio.h>
#include <math.h>
#include <cs50.h>
int main(void)
{
    float dollar;
    int quarters,dimes,nickles,pennies,cents,change,x;
// Loop to find the change
    do
    {
        dollar = get_float("change owed: ");
        cents = round(dollar * 100);
        quarters = cents / 25;
        cents = cents % 25;
        dimes = cents / 10;
        cents = cents % 10;
        nickles = cents / 5;
        cents = cents % 5;
        pennies = cents;
  // Total amount of change
        change = quarters + dimes + nickles + pennies;
    }
    while ( dollar < 0 );
    printf("%d\n",change);
}

File: Cash python/cash.py
from cs50 import get_float
while True:
    change = get_float("Change owed: ")
    if change >= 0:
        break

change *= 100

q = change//25
change %= 25
d = change//10
change %= 10
n = change//5
change %= 5

coins = q+d+n+change
coins = int(coins)
print(coins)
